tests/fixtures/.jscsrc

```diff
@@ -0,0 +1 @@
+{ "preset": "../../lib/jscsrc.json"}
```

Comment:
This lets JSCS find `lib/jscsrc.json` (which is exactly what a user would get if they did `preset: 'ember-suave'`).